modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SpringBootServerCodegen.java

```diff
@@ -119,9 +131,6 @@ public void addOperationToGroup(String tag, String resourcePath, Operation opera
         if (basePath == "") {
             basePath = "default";
         } else {
-            if (co.path.startsWith("/" + basePath)) {
-                co.path = co.path.substring(("/" + basePath).length());
-            }

```

Comment:
That looks like a change unrelated to the change in the PR description. What is its purpose?